wiki/Skinning/Tutorial/skin.ini/en.md

```diff
@@ -1,53 +1,34 @@
-# skin.ini
+# Guide: How to adjust my skin.ini
+
+*See also: [skin.ini](https://osu.ppy.sh/wiki/Skinning/skin.ini).*
+
+## What does the skin.ini do?
+The skin.ini determines how _osu!_ will treat some of the files of your skin.
+
+To create a `skin.ini` create a new textfile and name it `skin.ini`. These are the basic commands most skins use. Make sure that you type everything correctly and use a space ` ` after every `:`. *Remember, you aren't forced to add a value to every command! If you do leave one blank, remove it or comment it by prefixing with `//`.*
+
+### [General]
+`Name:` The name _osu!_ will give the skin when it gets ex-/imported as .osk.
+`Author:` The name of the creator = your name
+`Version:` Enter the newest version here, `2.5` at the release of this guide. You should never use `latest` here, this can cause bugs with newer versions when they get released.
+`AllowSliderBallTint:` If you want that the slider ball `sliderb` has the same color as your combo colour, type `1` in here. If you don't want this, type `0`.
+`CursorRotate:` If you want that your `cursor` rotates, type `1` in here. If you don't want this, type `0`.
+`CursorExpand:` If you want that your `cursor` gets bigger when you click, type `1` in here. If you don't want this, type `0 `.
+`CursorTrailRotate:` If you want that your `cursortrail` rotates, type `1` in here. if you don't want this, type `0`.
```

Comment:
`` `CursorTrailRotate:` Should the `cursortrail` rotate? If so, use `1`. if not, use `0`. ``